doc/man3/SM3.pod

```diff
@@ -0,0 +1,76 @@
+=pod
```

Comment:
Weren't you going to remove this file?

Reply:
Indeed, it's removed now. Just a reminder that with this change, now each EVP hash algorithm is going to need its own man file 😉